pygfx/materials/_compat.py

```diff
@@ -144,3 +144,132 @@ def material_from_trimesh(x):
         raise NotImplementedError(f"Conversion of {type(x)} is not supported.")
 
     return gfx_material
+
+
+def texture_from_numpy_array(image_array, dim=2, **kwargs):
```

Comment:
I don't get the use for this function. It does not seem to add anything that `gfx.Texture(image_array, dim=2)` already does, except converting to an ndarray when the input is an open3d image. I think this conversion is best solved in another way.

Reply:
Good point, I removed the method and directly create the texture from the numpy array.